remote.tf

```diff
@@ -1,7 +1,10 @@
 data "terraform_remote_state" "common" {
-  backend = "atlas"
+  backend = "remote"
 
   config = {
-    name = var.tf_remote_common
+    organization = split("/", var.tf_remote_common)[0]
```

Comment:
Maybe add a description to the `tf_remote_common` entry in the vars.tf file to mention the organization/name convention.

Reply:
I've been dragging my feet on documenting that. To date, we haven't done very well with documenting variables. I could easily find myself being drawn into that rabbit hole of writing documentation on ALL of the variables. I'll do this one at least.